templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/list_tags.html.twig

```diff
@@ -303,14 +312,15 @@
 
                             </tr>
                             {% else %}
-                                <tr>
+                                <tr data-cy="">
```

Comment:
why that?

Reply:
Sorry https://github.com/demos-europe/demosplan-core/pull/3109/commits/c6f2d1c3b7369800bcf80b4edd1fdb830f4c2ff8
That for if empty topic List item!